docs/openapi/components/schemas/common/ExtendedProduct.yml

```diff
@@ -0,0 +1,81 @@
+$linkedData:
+  term: ExtendedProduct
+  '@id': https://w3id.org/traceability#ExtendedProduct
+title: Extended Product
+description: Extended product information.
+type: object
+properties:
+  type:
+    type: array
+    readOnly: true
+    const:
+      - ExtendedProduct
+    default:
+      - ExtendedProduct
+    items:
+      type: string
+      enum:
+        - ExtendedProduct
+  origination:
+    title: Origination
+    description: The place of origin.
+    $ref: ./Place.yml
+    $linkedData:
+      term: origination
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#originLocation
+  supplier:
+    title: Supplier
+    description: The supplier for this product.
+    $ref: ./Organization.yml
+    $linkedData:
+      term: supplier
+      '@id': https://www.gs1.org/voc/manufacturer
+  htsCode:
+    title: HTS Code
+    description: The Harmonized Tariff Schedule (HTS) code.
+    type: string
+    $linkedData:
+      term: htsCode
+      '@id': https://vocabulary.uncefact.org/applicableRegulatoryProcedure
```

Comment:
seems like there is probably a better term for this, but I did not find one easily

Reply:
I can switch this to `applicableTax` ("A tax applicable to this header, line, subordinate line or payment trade settlement or cross-border regulatory procedure."), to be a bit more specific